app/extensions/string_extensions.rb

```diff
@@ -397,6 +404,7 @@ def tp_nodiv(sanitize = true)
   def tpl_nodiv(sanitize = true)
     Textile.textilize(self, true, sanitize)
```

Comment:
Should this have an `.html_safe`?  Or better yet, this should leverage the proposed `.textilize_safe`.